src/screen/AddAccountScreen/AddressPreview.tsx

```diff
@@ -28,8 +28,8 @@ function AddressInfoPreview(props: PropTypes) {
   const [account, setAccount] = useState<AccountInfo>();
   const [inProgress, setInProgress] = useState(false);
 
-  const {display, detail} = useAccountDetail(network?.key, address, api);
-  const identity = detail?.data;
+  const {data, isLoading} = useAccountIdentityInfo(address);
+  const identity = data?.registration;
 
   useEffect(() => {
     if (api) {
```

Comment:
This effect, the `account` state and the `api` are not needed anymore, we are getting the account info from the new hook

Reply:
I updated the PR and added a new hook called `useAccountInfo` here. problem is `identityInfo` ( Registration ) is different from `AccountInfo`. please take another look here.